linkml_runtime/utils/schemaview.py

```diff
@@ -442,7 +486,8 @@ def ordered(self, elements: ElementDict, ordered_by: OrderedBy | None = None) ->
             return self._order_inheritance(elements)
         if ordered_by is None or ordered_by in (OrderedBy.PRESERVE, OrderedBy.PRESERVE.value):
             return elements
-        raise ValueError(f"ordered_by must be in OrderedBy or None, got {ordered_by}")
+        msg = f"ordered_by must be in OrderedBy or None, got {ordered_by}"
+        raise ValueError(msg)
```

Comment:
Separate out error message composition from raising the error to keep the stack trace neat and tidy

see https://pypi.org/project/flake8-errmsg/ for more info on why this is the best thing since sliced bread